jina/parsers/helper.py

```diff
@@ -108,7 +108,7 @@ def start_section(self, heading):
 
     def _get_help_string(self, action):
         help_string = ''
-        if '%(default)' not in action.help:
+        if 'default' not in action.help:
```

Comment:
this line should be kept, it is a hack for correct sphinx-doc

Reply:
ok, i will find another solution to fix the help message for `--public/--private` exclusive argument.